cmd/influxd/config.go

```diff
@@ -251,6 +252,43 @@ func ParseConfig(s string) (*Config, error) {
 	return c, nil
 }
 
+type Graphite struct {
+	Addr          string `toml:"address"`
+	Database      string `toml:"database"`
+	Enabled       bool   `toml:"enabled"`
+	Port          uint16 `toml:"port"`
+	Protocol      string `toml:"protocol"`
+	NamePosition  string `toml:"name-position"`
+	NameSeparator string `toml:"name-separator"`
+}
```

Comment:
Can you rename this type to `graphite.Config`? A `Graphite` is ambiguous and `graphite.Graphite` is redundant.